tutorials/02a_RunningREMINDLocally.md

```diff
@@ -1,10 +1,146 @@
 # Running REMIND locally
-Mika Pflüger (mika.pflueger@pik-potsdam.de)
+Mika Pflüger (mika.pflueger@pik-potsdam.de), Tonn Rüter (tonn.rueter@pik-potsdam.de)
 
-*Note:* This tutorial is a work in progress, not all steps needed are described, yet.
+*Note:* This approach of running REMIND is a work in progress, however successful REMIND runs on local machines have been conducted using this tutorial.
+
+## System Requirements
+
+- Installed python version must be `3.10.X` or older. To check your python version run `python --version` in a terminal of your choice
+- GAMS must be installed and the path to GAMS applications needs to be in your PATH environment variable
+- R version must be at least `4.2.X`. To check your R version run `R --version` in a terminal of your choice
+- On Windows we recommend to use PowerShell rather than the standard command prompt
+
+## Use Local Input Data
+
+**This is the supported way of input data handling**
+
+As long as we do not provide a curated collection of input data please obtain these files from the cluster:
+
+```bash
+rev6.606_62eff8f7_remind.tgz
+rev6.606_62eff8f7_validationremind.tgz
+CESparametersAndGDX_878ac5d69254efb4eba5c1fa39aba64000307bb1.tgz
+```
+
+The contents of these tar-archives are necessary to run REMIND locally. Since REMIND input data is constantly updated the up-to-date file names may vary, but will exhibit a similar pattern. The first two files can be found on the cluster at `/p/projects/rd3mod/inputdata/output`, the latter file in `/p/projects/remind/inputdata/CESparametersAndGDX`.
```

Comment:
Mention also: Make sure that the configuration parameters `inputRevision` and `CESandGDXversion`  in your `config/default.cfg` match the file names.

Reply:
Done